src/assembler.m

```diff
@@ -136,24 +141,38 @@
     else
       error('wrong hyperElasModel for frame element.')
 		end
-    % -----------   aerodynmamic force   ------------------------------------
+    if ~isempty(elemTypeAero) &&  aeroBool== 0
```

Comment:
sugiero corregir indentación en los bloques de código agregados al assembler.m
sugiero usar comillas simples en los mensajes

Reply:
Indentation and spaces corrected in the hole script